subscription/protocol.go

```diff
@@ -0,0 +1,91 @@
+// Package subscription implements the Subscription NEX protocol
+package subscription
+
+import (
+	"fmt"
+
+	nex "github.com/PretendoNetwork/nex-go"
+	"github.com/PretendoNetwork/nex-protocols-go/globals"
+)
+
+const (
+	// SubscriptionProtocolID is the protocol ID for the Subscription protocol
+	SubscriptionProtocolID = 0x75
+
+	// SubscriptionCreateMySubscriptionDataID is the method ID for the method SubscriptionCreateMySubscriptionData
+	SubscriptionCreateMySubscriptionDataID = 0x1
+
+	// SubscriptionUpdateMySubscriptionDataID is the method ID for the method SubscriptionUpdateMySubscriptionData
+	SubscriptionUpdateMySubscriptionDataID = 0x2
+
+	// SubscriptionGetFriendSubscriptionDataID is the method ID for the method SubscriptionGetFriendSubscriptionData
+	SubscriptionGetFriendSubscriptionDataID = 0x7
+
+	// SubscriptionGetTargetSubscriptionDataID is the method ID for the method SubscriptionGetTargetSubscriptionData
+	SubscriptionGetTargetSubscriptionDataID = 0x8
+
+	// SubscriptionGetActivePlayerSubscriptionDataID is the method ID for the method SubscriptionGetActivePlayerSubscriptionData
+	SubscriptionGetActivePlayerSubscriptionDataID = 0x9
+
+	// SubscriptionGetSubscriptionDataID is the method ID for the method SubscriptionGetSubscriptionData
+	SubscriptionGetSubscriptionDataID = 0xA
+
+	// SubscriptionReplaceTargetAndGetSubscriptionDataID is the method ID for the method SubscriptionReplaceTargetAndGetSubscriptionData
+	SubscriptionReplaceTargetAndGetSubscriptionDataID = 0xb
+
+	// SubscriptionGetPrivacyLevelsID is the method ID for the method SubscriptionGetPrivacyLevels
+	SubscriptionGetPrivacyLevelsID = 0xf
+)
```

Comment:
Can you add [these](https://discord.com/channels/408718485913468928/881694543051907072/1138140868914724865) method constants aswell?